src/datreant/core/collections.py

```diff
@@ -620,6 +620,59 @@ def __getitem__(self, index):
             out = super(Bundle, self).__getitem__(index)
         return out
 
+    def get(self, *tags, **categories):
```

Comment:
I don't like the API. Without the unpacking it would be much clearer. We could then also allow tags to work with the new selection string parser

Reply:
how would that work without tags?  `bundle.get([], {'foo': 'bar'})`?

Reply:
Your API is fine. Not sure what I thought anymore. But the selection string parser will still work